api/external/compliance/reporting/reporting.swagger.json

```diff
@@ -45,6 +45,47 @@
         ]
       }
     },
+    "/api/v0/compliance/reporting/nodeheader/id/{id}": {
+      "post": {
```

Comment:
Modify as get method

Reply:
We need to use post as we are sending filter payload